lib/modules/manager/helmfile/extract.ts

```diff
@@ -18,7 +18,9 @@ export function extractPackageFile(
   let docs: Doc[];
   const aliases: Record<string, string> = {};
   try {
-    docs = loadAll(content, null, { json: true });
+    docs = loadAll(content.replace(/\s*({{.+}})\s*(\n)/gi, '$2'), null, {
```

Comment:
please extract a helper util function, so we can extend later.


```suggestion
    docs = loadAll(content.replace(/^[ \t]*{{.+}}[ \t]*(\n)$/gm, '$1'), null, {
```

your regex is too loose

- `\s` also capture newline
- no need for case insensitive
- use multiline flag, so we can limit to a single line

will create a regex101.com test tomorrow

Reply:
I took the liberty of applying your suggestions to a regex 101 [here](https://regex101.com/r/qs8new/1) but I think I may have errored in copying the regex over as it seems to match nothing (or I misconfigured the settings in regex101).

Reply:
Removing the beginning (`^`) and end of line (`$`) matchers does seem to do the trick though. Keeping them in and removing the newline capture `(\n)` will also work, but that will not replace templates within the line. For example, `version: {{ .Values.example.version }}` would not be changed to `version: `.

Reply:
What about this?
```suggestion
    docs = loadAll(content.replace(/(^|:)\s*{{.+}}\s*$/gm, ''), null, {
```
https://regex101.com/r/JSiGuZ/1